src/components/account/LeftMenu.jsx

```diff
@@ -0,0 +1,35 @@
+import { useAuth } from '../../context/AuthProvider';
+import { Button } from '@headlessui/react';
+
+import { useAccount } from '../../context/AccountProvider';
+import ChangePageButton from './ChangePageButton';
+
+const LeftMenu = () => {
+  const { clearUserSession } = useAuth();
+  const { setAccountPage } = useAccount();
+
+  return (
+    <div className="ml-5 hidden rounded-md border border-gray p-3 sm:block">
+      <Button
+        as="button"
+        onClick={() => setAccountPage('addBook')}
+        className="mb-6 block rounded-md bg-darkGreen px-3 py-1 text-white"
+      >
+        + Add New Listing
+      </Button>
+
+      <ChangePageButton page="myBooks" label="My Book Listings" />
```

Comment:
It would be nice if links sidebar nav links had some kind of on:hover animation.